packages/ui/src/components/va-accordion/VaAccordion.vue

```diff
@@ -35,14 +35,21 @@ export default defineComponent({
 .va-accordion {
   font-family: var(--va-font-family);
 
-  // Do not remove border radius from expanded and next to it collapse
-  .va-collapse:not(.va-collapse--expanded, .va-collapse--expanded + .va-collapse) {
+  .va-collapse {
     &:not(:first-child, :last-child) {
       .va-collapse__header__content {
         border-radius: 0;
       }
     }
 
+    &.va-collapse--expanded {
+      &:last-child {
+        .va-collapse__header__content {
+          border-radius: 0;
```

Comment:
For https://github.com/epicmaxco/vuestic-ui/issues/2021 we need simply remove margins in accordion.
See docs: https://vuestic.dev/en/ui-elements/accordion#default